packages/react-web/package.json

```diff
@@ -67,6 +69,7 @@
     "typescript": "^4.9.5"
   },
   "peerDependencies": {
+    "@xmtp/react-sdk": "1.0.0-preview.39",
```

Comment:
could be annoying as maybe you don't plan to use xmpt in your frontend but it would force you to anyway install it as a dependency in your project.

Reply:
shall I make it optional then?

```
"peerDependenciesMeta": {
  "@xmtp/react-sdk": {
    "optional": true
  },
},
```

Reply:
optional could work, let's try.